homework-classes/ContributorsView.js

```diff
@@ -19,8 +19,39 @@
      * @param {Object[]} contributors An array of contributor objects
      */
     render(contributors) {
-      // TODO: replace this comment and the console.log with your own code
-      console.log('ContributorsView', contributors);
+      this.container.innerHTML = '';
+      createAndAppend('p', this.container, {
+        text: `Contributors:`,
+        id: 'contributors-title',
+        role: 'heading',
+      });
+      const div = createAndAppend('div', this.container, {
```

Comment:
You seem to have missed the point that I made in slack about ARIA-compliance:

>For instance, for what is semantically supposed to be a list, a `ul` or `ol` element would be read out as a list. However, creating a list as a collection of `div` elements would not. In that case you would need to add ARIA attributes to the relevant HTML elements to help a screen reader recognize the semantics.

Reply:
Thank you so much for everything you've done over the past weeks. 
I've learned a lot from you!
I will continue to dig into the ARIA documentation until I find a way to use its attributes correctly.

Reply:
Glad to hear, Rabie 👍.

For ARIA-compliance in this case, you only need to replace your nested divs with a `ul` with `li` children. This is semantically a list that is ARIA-compliant. A screen reader will identify that as a list. But it would not know what to make of a `div` unless you add ARIA attributes to explain its role. But why choose that over a `ul`?

Reply:
I thought I shouldn't replace the divs with `ul` and `li `  in the borrowed snippets. Because of that I was looking for alternative attributes to identify my divs. 

Well, this is what beginners would mistakenly do :)

I will update the work